A41-xds-rbac.md

```diff
@@ -0,0 +1,199 @@
+A41: xDS RBAC Support
+----
+* Author(s): [Eric Anderson](https://github.com/ejona86), [Ashitha
+  Santhosh](https://github.com/ashithasantosh) (RBAC engine behavior)
+* Approver: markdroth
+* Status: Draft {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: <language, ...>
+* Last updated: 2021-05-07
+* Discussion at: <google group thread> (filled after thread exists)
+
+## Abstract
+
+Support the [xDS RBAC HTTP filter][RBAC filter] for service- and method-scoped
+client authorization (authz) on xDS-enabled gRPC servers.
+
+[RBAC filter]: https://github.com/envoyproxy/envoy/blob/main/api/envoy/extensions/filters/http/rbac/v3/rbac.proto
+
+## Background
+
+[A29 xDS-Based Security][A29] introduced the ability to manage xDS-managed mTLS
+meshes. As standard for TLS clients, the client verifies the server's identity
+is permitted to run the service. This is normally known as "hostname
+verification", but in a mesh scenario may be known as "server authorization" as
+the server's certificate does not include the hostname but other identity
+information.
+
+While servers authenticated clients (i.e., the client's identity was verified),
+A29 did not include mechanism to limit which clients were authorized to access a
+server and for which gRPC methods (i.e., whether the client was permitted).
+
+Envoy defines the [RBAC HTTP filter][RBAC filter] that is well-suited for
+limiting which nodes can access which services within a mesh as encouraged by
+the principal of least privilege.
+
+Resources in REST are present in the HTTP path so it would seem RBAC could be
```

Comment:
The phraseology in this paragraph is a little cumbersome, making it a little hard to understand.  I think what you're referring to is the fact that we won't be able to support authz based on message payloads?  If so, maybe just say that?

Reply:
It's a bit more "gRPC puts something in a message payload that REST does not." I'll work on the language.

Reply:
@markdroth, I've reworded this paragraph. The paragraph is saying "you can't look at message payloads" but also "that isn't a big deal, as it isn't all that useful in REST."